### PR TITLE
Support Cancun and Prague revisions in state tests

### PR DESCRIPTION
**Pull request by gumb0**

### CODE REVIEW

**Comment by gumb0** [comment]
Probably better to add Prague right away too, but that requires change in EVMC.

**Comment by axic** [comment]
> Probably better to add Prague right away too, but that requires change in EVMC.

Agree, let's do that.

File: lib/evmone/instructions_traits.hpp [hunk 1]
```diff
@@ -167,6 +167,8 @@ constexpr inline GasCostTable gas_costs = []() noexcept {
     table[EVMC_CANCUN][OP_DUPN] = 3;
     table[EVMC_CANCUN][OP_SWAPN] = 3;
 
+    table[EVMC_PRAGUE] = table[EVMC_CANCUN];
```
Comment (axic) [inline lib/evmone/instructions_traits.hpp:170]: Do you want to extend unit tests for this?

**Review by axic — APPROVED** [review]

File: test/integration/CMakeLists.txt [hunk 2]
```diff
@@ -22,7 +22,7 @@ DUP1,4
     add_test(NAME ${PREFIX}/trace COMMAND $<TARGET_FILE:evmc::tool> --vm $<TARGET_FILE:evmone>,trace run 60006002800103)
     set_tests_properties(
         ${PREFIX}/trace PROPERTIES PASS_REGULAR_EXPRESSION
-        "{\"depth\":0,\"rev\":\"London\",\"static\":false}
+        "{\"depth\":0,\"rev\":\"Paris\",\"static\":false}
```
Comment (axic) [inline test/integration/CMakeLists.txt:25]: Maybe merge this commit into the Prague commit and extend the commit log with "Additionally EVMC set the latest stable revision to Paris".

**Review by axic — APPROVED** [review]

**Pull request merged**